src/status_im2/common/standard_authentication/enter_password/view.cljs

```diff
@@ -32,20 +32,21 @@
           :customization-color customization-color
           :size                24}]]
        [password-input/view
-        {:processing       processing
-         :error            error
-         :default-password password
-         :sign-in-enabled? sign-in-enabled?}]
-       [rn/view {:style style/enter-password-button}
-        [quo/button
-         {:size                40
-          :type                :primary
-          :customization-color (or customization-color :primary)
-          :accessibility-label :login-button
-          :icon-left           button-icon-left
-          :disabled?           (or (not sign-in-enabled?) processing)
-          :on-press            (fn []
-                                 (rf/dispatch [:set-in [:profile/login :key-uid] key-uid])
-                                 (rf/dispatch [:profile.login/verify-database-password password
-                                               #(on-enter-password password)]))}
-         button-label]]]]]))
+        {:on-press-biometrics on-press-biometrics
+         :processing          processing
+         :error               error
+         :default-password    password
+         :sign-in-enabled?    sign-in-enabled?}]
+       [quo/button
+        {:size                40
+         :container-style     style/enter-password-button
+         :type                :primary
+         :customization-color (or customization-color :primary)
+         :accessibility-label :login-button
+         :icon-left           button-icon-left
+         :disabled?           (or (not sign-in-enabled?) processing)
+         :on-press            (fn []
+                                (rf/dispatch [:set-in [:profile/login :key-uid] key-uid])
+                                (rf/dispatch [:profile.login/verify-database-password password
+                                              #(on-enter-password password)]))}
```

Comment:
Would it make sense to have these 2 events into one, or is it better to keep them separated?